client/test/cypress/support/commands.js

```diff
@@ -12,12 +12,12 @@ Cypress.Commands.add("dataCy", value => {
 
 /**
  * Login to the api without providing authentication.
- * 
+ *
  * @example cy.login({email: 'mytestuser@example.com'})
  */
 Cypress.Commands.add('login', ({ email }) => {
   cy.xsrfToken().then((token) => {
-    
+
```

Comment:
I've got to check my settings and figure out why I'm leaving trailing garbage all over the place.